reloj-checador/app/models/employee.rb

```diff
@@ -0,0 +1,3 @@
+class Employee < ApplicationRecord
+  belongs_to :company
+end
```

Comment:
I recommend that you perform validations on the models, to prevent the end user from making empty employees. 

![imagen](https://user-images.githubusercontent.com/12243331/106797367-1dcfa200-6622-11eb-8f83-bd257f4fb64c.png)